vision/tools/blob_annotator/main.py

```diff
@@ -237,7 +237,7 @@ def on_mouse_event(self, event, x, y, flags, params):
 
 
 if __name__ == '__main__':
-    DATASET_PATH = os.path.join("..", "..", "vision_images", "obstacle")
+    DATASET_PATH = os.path.join("../../vision_images/boat/")
```

Comment:
This should be text. Make sure to update the folder name of the dataset in the google drive.